modules/ROOT/pages/munit-extensions-maven-plugin-configuration.adoc

```diff
@@ -0,0 +1,534 @@
+        <latestPatches>true</latestPatches>
+    </discoverRuntimes>
+</runtimeConfiguration>
+----
+
+From the command line, the property is `discoverRuntimes.latestPatches`. The default value is `true`.
+
+=== Discover Since a Minimum Version
+
+By default, runtime discovery searches for all Mule runtime versions since the `minMuleVersion` of the module. To change this minimum version, add the following line to the runtime configuration:
+
+[source,xml,linenums]
+----
+<runtimeConfiguration>
+    <discoverRuntimes>
+        <product>CE</product>
+        <minMuleVersion>4.1.2</minMuleVersion>
+    </discoverRuntimes>
+</runtimeConfiguration>
+----
+
+From the command line, the property is `discoverRuntimes.minMuleVersion`. The default value is the `minMuleVersion` of the module.
+
+=== Run Against Additional Runtimes
+
+By default, the MUnit Extensions Maven plugin runs against the `minMuleVersion` and the required product of the module. To specify additional runtimes to run against, add the following lines to the runtime configuration:
+
+[source,xml,linenums]
+----
+<runtimeConfiguration>
+    <additionalRuntimes>
+        <additionalRuntime>MULE_EE:4.1.2</additionalRuntime>
+        <additionalRuntime>MULE:4.1.1</additionalRuntime>
+    </additionalRuntimes>
+</runtimeConfiguration>
+----
+
+From the command line, the property is `additionalRuntimes`.
+
+=== Skip Runtime Discovery
+
+To skip runtime discovery, add the following line to the runtime configuration:
+
+[source,xml,linenums]
+----
+<runtimeConfiguration>
+    <discoverRuntimes>
+        <product>CE</product>
+        <skip>true</skip>
+    </discoverRuntimes>
+</runtimeConfiguration>
+----
+
+From the command line, the property is `discoverRuntimes.skip`. The default value is `false`.
+
+== Use Dynamic Ports
+
+[IMPORTANT]
+MUnit 2.2 and later introduces the `dynamic-port` global element, which you can use to define dynamic ports at the MUnit suite level. +
+Use this element instead of the plugin configuration described below to set the dynamic port both from Maven and Studio. +
+For more information about how to configure this element, refer to xref:munit::dynamic-ports.adoc[Dynamic Ports].
+
+When testing a module in a continuous integration (CI) environment, you might encounter the following scenario:
+
+`Your test tries to open a specific port. The port is already in use. The test fails with a port binding exception.`
+
+To solve this, use the dynamic ports built-in feature in the MUnit Extensions Maven plugin to have your test use a free port. Dynamic ports instruct the MUnit Extensions Maven plugin to look for unbound ports and reserves them before running the tests over the module. Each port selected is placed in a system property under the name indicated in the configuration. The test can acquire the port number through the use of placeholders afterward.
+
+TIP: The ports to select by the MUnit Extensions Maven plugin are taken from the following range: `[40000,50000)`.
```

Comment:
Does this need to be a TIP? It is not a good  practice to have two admonitions in a row.